components/org.wso2.carbon.status.dashboard.web/src/add-worker/AddWorker.jsx

```diff
@@ -339,7 +336,7 @@ export default class AddWorker extends React.Component {
                 } else {
                     this.setState({
                         isApiCalled: true,
-                        statusMessage: "Unknown error occurred!"
+                        statusMessage: "Unknown error occurred4444!"
```

Comment:
Is this intended or typo?

Reply:
Fixed in https://github.com/wso2/carbon-analytics/pull/1256